AugurProject/augur
# fix claim reporting fees modal updating issue

fixed issue where claim reporting fees modal doesn't update if user claims individual line item or when claiming individual item is the last thing claimable.


fixes <https://github.com/augurproject/augur/issues/4231>

to test: 

use fake time
1. buy participation tokens
2. push time a week
3. buy participation tokens
4. push time a week
5. buy participation tokens again or create a market (need event to create new dispute window to make sure SDK is in accurate current state)

you should get a notification in account summary page. 
`claim` participation tokens. sign MM transaction and modal should close, since there isn't anything else to claim.
